debian/copyright

```diff
@@ -0,0 +1,51 @@
+Format: https://www.debian.org/doc/packaging-manuals/copyright-format/1.0/
+Upstream-Name: sosreport
+Upstream-Contact: Bryn M. Reeves <bmr@redhat.com>
```

Comment:
This should probably be me, if anyone specific. Feel free to put my personal email here (see GH profile).

Reply:
updated